### DISCUSSION TITLE
Feat: Better horizontal scroll

### DISCUSSION THREAD

**Discussion by TizzySaurus** [aws/codewhisperer-command-line-discussions#108] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
The CW popup when typing a command allows you to scroll horizontally on the command description to see the full description. It would be great if you could do the same for the suggestions, e.g. in the below video to see the full name of the `a-fake-really-long-directory-name-to-showcase-an-example` folder.

I'm aware that you can change the width of the popup window in the settings, but that's not ideal because you're then constantly increasing the width of the popup, even when that extra space isn't needed (which from my experience, is most of the time).

An alternative would be to allow us to resize the popup window by clicking and dragging and the edge of the popup window, but I'd prefer the scrolling option.

A more realistic use-case is for something like zoxide/`z`, where the autocomplete shows the full path which is almost always too long (3 here are cut-short, so I can't actually see where `z` is trying to take me): 
<img width="633" alt="image" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/47674925/264c76ef-0126-4439-a911-9c2122d5204d">

Video showcasing that you can horizontally scroll the command description, but not the suggestions:

https://github.com/aws/codewhisperer-command-line-discussions/assets/47674925/de82b6ed-4fec-4612-a17c-4e3be97aafcc

**Comment by brendanfalk** [comment]
Nice catch! I believe we actually had this with Fig (in fact we had it auto-scroll) but clearly that didn't port over nicely to CodeWhisperer. We will fix!

**Reply by TizzySaurus** [reply]
Great, thanks!
